tests/transforms/core/test_transform_program.py

```diff
@@ -356,6 +356,13 @@ def test_empty_program(self):
         ):
             program.get_last()
 
+    def test_get_last(self):
```

Comment:
Was this just... not tested?

Reply:
Yeah but for some reason codecov never complained before and complains now.

Reply:
😢